build/dockerfiles/Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM debian:bullseye@sha256:71cb300d5448af821aedfe63afd55ba05f45a6a79f00dcd131b96b780bb99fe4
+
+ENV CORE_RS_ALBATROSS_COMMIT_HASH=93066288a6bfccc43498ae5596cdf2189b244785
+ENV CORE_RS_ALBATROSS_DIGEST=f606933de4341821be1ab765167c2637c20afb0b50b889b4311242f4a407259d
+
+# Set system time to 00:00:01 UTC
+ENV SOURCE_DATE_EPOCH=1
+
+# Avoid prompts during installation
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Copy apt install files from repo
+COPY apt/packages/ ./
+
+# Copy digest for apt install files
+COPY apt/apt.lockfile ./
+
+# Install all deb files
+RUN dpkg -i *.deb || true
+
+# Create nimiq directory for build
+RUN mkdir -p /home/nimiq/
+
+# Change homedir
+WORKDIR /home/nimiq
+
+# Clone core-rs-albatross repo
+RUN curl -LO https://github.com/nimiq/core-rs-albatross/archive/${CORE_RS_ALBATROSS_COMMIT_HASH}.zip \
+    && [ "$(sha256sum ${CORE_RS_ALBATROSS_COMMIT_HASH}.zip | awk '{print $1}')" = "$CORE_RS_ALBATROSS_DIGEST" ] \
+    && unzip ${CORE_RS_ALBATROSS_COMMIT_HASH}.zip -d /home/nimiq \
+    && mv core-rs-albatross-${CORE_RS_ALBATROSS_COMMIT_HASH} core-rs-albatross/ \
+    && rm ${CORE_RS_ALBATROSS_COMMIT_HASH}.zip
+
+# Set workdir to source 
+WORKDIR /home/nimiq/core-rs-albatross
+
+# Install rustup
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+
+# Set environment variables for Rust
+ENV PATH="/root/.cargo/bin:${PATH}"
+
+# Lock the rust toolchain
+RUN rustup toolchain install 1.50.0
```

Comment:
Out of curiosity, why going for toolchain 1.50?

Reply:
It's just to pin it, we can bump it to a higher version.

Reply:
I'd suggest to move to latest stable: 1.75.0